.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,37 @@
+
+name: update-version
+on:
+  pull_request:
+          types: [opened]
+          branches: [master]
```

Comment:
Hmm so this means it'll run just on open? What happens when release please has to rerun? 

I think we should restrict this based on `[paths](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths)` as well - we only want this to run if version.txt has changed.

Reply:
I agree on the path restriction. I forgot that release please closes and reopens the PR every time it edits it.